main.go

```diff
@@ -156,10 +156,10 @@ func NewGame(g *Game) {
 			Y: (pathCoord.(map[string]any)["cy"].(float64) + 0.5) * float64(entities.GridSize),
 		}
 	}
-
+	
 	// Add dog to the game
 	g.Dog = &Dog{
-		Object:   resolv.NewObject(float64(dogEntity.Position[0]), float64(dogEntity.Position[1]), 16, 16, tagDog),
+		Object:   resolv.NewObject(float64(dogEntity.Position[0]), float64(dogEntity.Position[1]), 8, 8, tagDog),
```

Comment:
If you set the object size to 8x8 and don't alter the position of either it or the sprite then AFAIK instead of colliding with the centre, other objects will collide with the dog's top-left 8x8 quarter, which is probably not what you want.

Reply:
Center of the dog sprite is drawn to the center of the Object, so it seems it collides with the center of the dog. I would keep it until we have a proper solution